File: 1045.cpp
#include <algorithm>
#include <iostream>
using namespace std;

int main() {
    double A, B, C;

    cin >> A >> B >> C;
    double v[] = {A, B, C};
    std::sort(v, v + 3);
    A = v[2], B = v[1], C = v[0];

    if(A >= B + C) {
        cout << "NAO FORMA TRIANGULO\n";
    }else {
        if(A * A == B * B + C * C)
            cout << "TRIANGULO RETANGULO\n";
        else if(A * A > B * B + C * C)
            cout << "TRIANGULO OBTUSANGULO\n";
        else
            cout << "TRIANGULO ACUTANGULO\n";

        if(A == B && A == C)
            cout << "TRIANGULO EQUILATERO\n";
        else if(A == B || B == C|| A == C)
            cout << "TRIANGULO ISOSCELES\n";
    }

    return 0;
}

File: 1045.java
import java.util.Arrays;
import java.util.Collections;
import java.util.Scanner;
import java.util.Locale;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in).useLocale(Locale.US);
		
		double a, b, c;
		a = sc.nextDouble();
		b = sc.nextDouble();
		c = sc.nextDouble();
		
		Double[] v = {a, b, c};
		Arrays.sort(v, Collections.reverseOrder());
		
		a = v[0];
		b = v[1];
		c = v[2];		
		
		if(a >= (b + c)) {
	        System.out.println("NAO FORMA TRIANGULO");
	    }else {
	        if((a * a) == (b * b) + (c * c))
	        	System.out.println("TRIANGULO RETANGULO");
	        else if((a * a) > (b * b) + (c * c))
	        	System.out.println("TRIANGULO OBTUSANGULO");
	        else if((a * a) < (b * b) + (c * c))
	        	System.out.println("TRIANGULO ACUTANGULO");
	        if(a == b && b == c)
	        	System.out.println("TRIANGULO EQUILATERO");
	        else if (a == b || a == c || b == c)
	        	System.out.println("TRIANGULO ISOSCELES");
	    }
			
		sc.close();
	}
}
